### PR TITLE
lepley_vatrp768

### PR DESCRIPTION
**Pull request by ladyhawk**
Liz E.: adding linphone lib version info to logs. Logged on launch with ACE iPhone version, as well.

### CODE REVIEW

File: linphone-Info.plist [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.17</string>
+	<string></string>
```
Comment (kennethdick) [inline linphone-Info.plist]: Why are you removing version?

File: submodules/linphone [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit f8b863e536a683cb44145ff1132459e51fb4e0c4
+Subproject commit 73c6b9e74516ca8acbd6d1432522f46f936e61e0
```
Comment (kennethdick) [inline submodules/linphone]: revert this change, we want latest
https://github.com/VTCSecureLLC/linphone/commit/f8b863e536a683cb44145ff1132459e51fb4e0c4

Comment (ladyhawk) [inline submodules/linphone]: ok

File: linphone-Info.plist (hunk 1 above)
Comment (ladyhawk) [inline linphone-Info.plist]: hm. that was not intentional - fixing it.

**Comment by ladyhawk** [comment]
REverted two noted changes.

**Comment by kennethdick** [comment]
+1
